sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/autoconfigure/b2c/AADB2CAuthorizationRequestResolver.java

```diff
@@ -59,13 +60,17 @@ public OAuth2AuthorizationRequest resolve(@NonNull HttpServletRequest request) {
 
     @Override
     public OAuth2AuthorizationRequest resolve(@NonNull HttpServletRequest request, String registrationId) {
-        if (StringUtils.hasText(passwordResetUserFlow) && isForgotPasswordAuthorizationRequest(request)) {
-            final OAuth2AuthorizationRequest authRequest = defaultResolver.resolve(request, passwordResetUserFlow);
-            return getB2CAuthorizationRequest(authRequest, passwordResetUserFlow);
-        }
+        if (StringUtils.hasText(registrationId)) {
+            if (REQUEST_MATCHER.matches(request)) {
+                return getB2CAuthorizationRequest(defaultResolver.resolve(request), registrationId);
+            }
+
+            if (isForgotPasswordAuthorizationRequest(request)) {
```

Comment:
should we also specify which user flow is passwordResetUserFlow here?

Reply:
that can only be specialized by adding an explicit attribute in AADB2CProperties, so shall we do this?